### ISSUE TITLE
VictoryAnimation throws error in IE10

### ISSUE THREAD

**Issue by okonet** [FormidableLabs/victory#248]
I'm getting `SCRIPT5007: Unable to get property 'data' of undefined or null reference` 

in the following line

``` js
return n.state=Array.isArray(n.props.data)?n.props.data[0]:n.props.data
```

in latest IE10 for the following code:

``` jsx
            <VictoryAnimation
                data={{ medianY }}
                duration={250}
                easing="elasticInOut"
            >
                {
                    props =>
                        <g>
                            <line
                                x1={PADDING.left}
                                x2={width}
                                y1={props.medianY}
                                y2={props.medianY}
                                className={styles.medianLine}
                            />
                            <text
                                x={width - 7}
                                y={props.medianY - 3}
                                className={styles.medianValue}
                            >
                                <tspan>~</tspan>
                                <FormattedDuration
                                    svg
                                    duration={medianResponseTime}
                                />
                            </text>
                        </g>
                }
            </VictoryAnimation>
```

The same code works fine in IE11.

**Comment by flipace** [comment]
We experience exactly the same issue, although when using VictoryPie with some ```animate``` property object.

**Comment by jmcriffey** [comment]
Here is a possible lead: https://phabricator.babeljs.io/T3041. I've had to deal with a similar issue in a large code base. It seems that Babel officially unofficially dropped all IE support in Babel 6, and it can cause trouble with class constructors in older browsers. I ended up having to use loose class mode and the proto-to-assign plugin to get the code base 100% working in IE9 and IE10. 

I can't seem to remember what was actually causing the issue, as only some classes were causing issues. My case had something to do with the base class the setup in its constructor.
